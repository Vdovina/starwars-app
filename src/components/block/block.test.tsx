import { create, act as testAct, ReactTestRenderer } from 'react-test-renderer';
import { Block } from './';

describe('Block Tests', () => {
  it('Block render', () => {
    let component: ReactTestRenderer;
    testAct(() => {
      component = create(<Block title="BlockTitle">ChildrenNode</Block>);
    });
    expect(component.toJSON()).toMatchSnapshot();
  });
});
