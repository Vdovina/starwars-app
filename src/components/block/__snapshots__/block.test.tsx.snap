// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Block Tests Block render 1`] = `
<div
  className="block"
>
  <div
    className="block__title"
  >
    BlockTitle
  </div>
  ChildrenNode
</div>
`;
